package.json

```diff
@@ -63,7 +63,7 @@
     "prop-types": "^15.8.0",
     "react-inlinesvg": "^2.3.0",
     "react-popper": "^2.2.5",
-    "react-select": "^3.1.0",
+    "react-select": "^4.0.0",
```

Comment:
as far as i seen they have only 1 big change which not relevant to the way we implement our drop down - the change is in the on change function 1 param value: https://react-select.com/upgrade#from-v3-to-v4